cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections
+from psycopg2 import OperationalError
```

Comment:
Consider importing `OperationalError` from `django.db.utils` instead of `psycopg2`. This ensures compatibility with different database backends supported by Django, not just PostgreSQL.